pages/common/rails-generate.md

```diff
@@ -0,0 +1,19 @@
+# rails generate
+
+> Generate new rails templates in an existing project.
```

Comment:
Does Rails usually start with a capital? i.e.
```suggestion
> Generate new Rails templates in an existing project.
```

Reply:
Yes, it does! Thanks for catching that.